client-bagel-buddy/src/components/card/index.js

```diff
@@ -6,7 +6,7 @@ function Card() {
     return (
         <div className='card'>
             <div className='card-img'
-                style={{ backgroundImage: `url(${user_img_url})` }}/>
+                style={{ backgroundImage: `url(${user_img_url})`, backgroundSize: 'cover'}}/>
```

Comment:
Nice! We do need to set background size to `cover`. let's put it in the `card-img` class in the css file